README.md

```diff
@@ -61,6 +61,7 @@ Commands:
     -p, --password=PASSWORD  Password for PKCS12/JCEKS key stores (reads from TTY if missing).
     -m, --pem                Write output as PEM blocks instead of human-readable format.
     -j, --json               Write output as machine-readable JSON format.
+    -d, --depth=0            Certificate information upto a certain depth.
```

Comment:
I wonder if `certificate chain` would be more clear here.

Reply:
Sure thing, I'll make the change and update the PR.